native/src/modules/common/components/TimeStamp.js

```diff
@@ -29,18 +29,19 @@ type PropsType = {|
   formatter: DateFormatter,
   t: TFunction,
   language: string,
-  theme: ThemeType
+  theme: ThemeType,
+  useText?: boolean
 |}
 
 export class TimeStamp extends React.PureComponent<PropsType> {
   render () {
-    const { lastUpdate, formatter, t, language, theme } = this.props
+    const { lastUpdate, formatter, t, language, theme, useText } = this.props
```

Comment:
```suggestion
    const { lastUpdate, formatter, t, language, theme, useText = true } = this.props
```
With this default value you could just use `useText && <TimeStampText ... ` below.